modules/gapi/perf/common/gapi_render_perf_tests_inl.hpp

```diff
@@ -0,0 +1,918 @@
+// This file is part of OpenCV project.
```

Comment:
You don't need this file in the `common` part, because it can only be used by one backend
You can put the test definition and its instance together in the same file (as for unit tests)